src/clientSideScene/sceneEntities.ts

```diff
@@ -1286,6 +1441,14 @@ function mouseEnterLeaveCallbacks() {
         sceneInfra.highlightCallback([node.start, node.end])
         const yellow = 0xffff00
         colorSegment(selected, yellow)
+        const extraSegmentGroup = parent.getObjectByName(EXTRA_SEGMENT_HANDLE)
+        if (extraSegmentGroup) {
+          extraSegmentGroup.traverse((child) => {
+            if (child instanceof Points || child instanceof Mesh) {
+              child.material.opacity = dragSelected ? 0 : 1
```

Comment:
Don't show the `+` handle if you're already dragging yeah?

Reply:
Yup that's it.